resources/dicts/patrols/general/hunting.json

```diff
@@ -3691,5 +3691,241 @@
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": -5
+		}
+	    ]
+        },
+	{
+            "text": "Spotting a rabbit, the bickering duo glance at each other, realising they'll need to work together to catch it. Feeding the Clan comes before rivalries. Putting their differences aside, r_c chases the rabbit while p_l gets into position, catching the rabbit when r_c drives it into {PRONOUN/r_c/poss} paws.",
+            "exp": 20,
+            "weight": 20,
+	    "prey": ["medium"],
+	    "relationships": [
+		{
+            	"cats_to": ["p_l"],
+            	"cats_from": ["r_c"],
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": -10
+		},
+	        {
+            	"cats_to": ["p_l"],
+            	"cats_from": ["r_c"],
+            	"mutual": true,
+            	"values": ["trust", "respect"],
+            	"amount": 10
+		}
+	    ],
+	    "art": "gen_hunt_rabbit"
+        }
+    ],
+    "fail_outcomes": [
+        {
+            "text": "The pair manage to keep the hunting patrol relatively on track, until s_c bursts into laughter after p_l slips and falls during a catch. Furious and <i>very</i> close to clawing s_c's obnoxious face off, p_l storms off back to camp.",
+            "exp": 0,
+            "weight": 20,
+	    "prey": ["very_small"],
+	    "stat_trait": ["childish", "oblivious", "strange", "troublesome"],
+	    "can_have_stat": ["r_c"],
+	    "relationships": [
+		{
+            	"cats_to": ["p_l"],
+            	"cats_from": ["s_c"],
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": 10
+		}
+	    ]
+            
+        },
+	{
+            "text": "{PRONOUN/p_l/poss} nerves already frayed, it only takes one more snide comment from r_c for p_l to explode, snarling in r_c's face the first insult that comes to mind. The two abruptly part ways, their hunting forgotten.",
+            "exp": 0,
+            "weight": 20,
+	    "prey": ["very_small"],
+	    "relationships": [
+		{
+            	"cats_to": ["p_l"],
+            	"cats_from": ["r_c"],
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": 10
+		}
+	    ]
+            
+        },
+	{
+            "text": "Already on edge from the constant bickering, when r_c makes a particularly hurtful remark, s_c retaliates, {PRONOUN/s_c/poss} claws scratching r_c's cheek. The pair silently return to camp - each cat seething - and refuse to tell any cat what happened.",
+            "exp": 0,
+            "weight": 20,
+	    "stat_trait": ["bloodthirsty", "fierce", "vengeful"],
+	    "relationships": [
+		{
+            	"cats_to": ["r_c"],
+            	"cats_from": ["s_c"],
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": 15
+		}
+	    ],
+	    "injury": [
+                {
+                  "cats": ["r_c"],
+                  "injuries": ["claw-wound"],
```

Comment:
claw-wound is a relatively major injury for how the text seems to describe this altercation, perhaps scrapes or torn pelt would be more apt?  scar could stay the same, as that's an interesting way for the cat to gain that particular scar.  Alternatively, adjust the outcome text to more appropriately mirror the more major claw-wound injury.